pkg/rehearse/jobs_test.go

```diff
@@ -361,8 +386,8 @@ func makeTestData() (int, string, string, *pjapi.Refs) {
 	testPrNumber := 123
 	testNamespace := "test-namespace"
 	testRefs := &pjapi.Refs{
-		Org:     "testRepo",
```

Comment:
Yikes -- what happened here?

Reply:
That was an old typo

Reply:
Did you change both ways? Why wasn't this test broken before or after?

Reply:
I guess we didn't even use that.... I will check why it wasn't failing...

Reply:
Well, we used those refs to pass them in the `Executor`, but we didn't evaluate the actual refs but only the execution. We could always extend the test but I think that they are ok as they are now.